src/Java.Interop.Tools.Maven/Models/Project.cs

```diff
@@ -0,0 +1,6589 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// https://github.com/mganss/XmlSchemaClassGenerator
+// This code was generated by XmlSchemaClassGenerator version 2.1.1057.0 using the following command:
+// xscgen maven-4.0.0.xsd --nullable --nc --nr
```

Comment:
Where is `maven-4.0.0.xsd`?  Should that be added to this project for "reference" purpsoes?

Reply:
I don't think we should add it to our repo, but adding a URL for it to the comment seems like a good idea.

Reply:
Given that this is a generated file, should we *move* this comment block & related into `Project.Partial.cs`, so that we don't have to manually update this file whenever we regenerate it?

Reply:
Relatedly, I imagine you didn't build mganss/XmlSchemaClassGenerator yourself, but instead used a NuGet package such as https://www.nuget.org/packages/XmlSchemaClassGenerator.Console.  Would be useful to put this into a `.targets` file to make updating easier.